sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/models/ece-integration.model.ts

```diff
@@ -0,0 +1,48 @@
+import { DisbursementValue } from "@sims/sims-db";
+
+export const ECE_SENT_TITLE = "CONFIRMATION REQUEST";
+
+/**
+ * Result of a file uploaded to SFTP on ZONE B network.
+ */
+export interface ECEUploadResult {
+  generatedFile: string;
+  uploadedRecords: number;
+}
+
+/**
+ * Represents a single line in a ECE request file.
+ * When implemented in a derived class this
+ * interface allow the object to be represented
+ * as a formatted fixed string.
+ */
+export interface ECERequestFileLine {
+  getFixedFormat(): string;
+}
+
+export interface ECERecord {
+  institutionCode: string;
+  disbursementValues: DisbursementValue[];
+  sin: string;
+  studentLastName: string;
+  studentGivenName: string;
+  birthDate: string;
+  sfasApplicationNumber: string;
+  institutionStudentNumber: string;
+  studyStartDate: string;
+  studyEndDate: string;
+  disbursementDate: string;
+}
+
+export const DATE_FORMAT = "YYYYMMDD";
+export const SPACE_FILLER = " ";
+export const NUMBER_FILLER = "0";
```

Comment:
If the space filler and number filler is not just specific to this integration, it can be a common constant at integration scope.

Reply:
If they are common to the institution's integrations I would be ok to share these.
Beyond that, the idea was to not share across integrations, for instance between ESDC and Institutions.

Reply:
Resolving as understood that the space and number filler can be unique requirements at file/client level.